tasks/uk/census/metadata.py

```diff
@@ -0,0 +1,89 @@
+from collections import OrderedDict
+import json
+import os
+import re
+
+from tasks.meta import OBSColumn, DENOMINATOR, OBSTag
+from tasks.util import ColumnsTask, TagsTask
+from tasks.tags import UnitTags, SectionTags, SubsectionTags, LicenseTags
+
+
+def load_definition():
+    with open(os.path.join(os.path.dirname(__file__), 'columns.json')) as json_file:
+        return json.load(json_file, object_pairs_hook=OrderedDict)
+
+
+COLUMNS_DEFINITION = load_definition()
+
+
+DISALLOWED_CHARACTERS_RE = re.compile(r'[:/, \-\.\(\)]')
+
+
+def sanitize_identifier(colid):
+    return DISALLOWED_CHARACTERS_RE.sub('_', '_'.join(colid.split(';')[0].split(':')[-2:]))
+
+
+def parse_table(table_id):
+    '''
+    Returns a tuple like (KS, NI) from a table string like KS201NI
+    '''
+    return re.match('([A-Z]{2})\d+([A-Z]{2})[a-z]?', table_id).groups()
+
+
+class SourceTags(TagsTask):
```

Comment:
As a reminder for a future refactor, we should also make the tags creation declarative